pytket/tests/classical_test.py

```diff
@@ -1115,178 +1115,6 @@ def check_serialization_roundtrip(circ: Circuit) -> None:
     assert circ_from_dict.to_dict() == circ_dict
 
 
-def test_decomposition_known() -> None:
```

Comment:
Next step after this will be to write a new decomposition pass, at which point I'll add a corresponding test to replace this one.